src/notifications/api/viewsets.py

```diff
@@ -68,9 +68,14 @@ class NotificatieViewSet(viewsets.ViewSet):
         'update': SCOPE_NOTIF_CHANGE_ALL,
         'partial_update': SCOPE_NOTIF_CHANGE_ALL,
     }
+    action = 'create'
 
     def create(self, request, *args, **kwargs):
-        serializer = self.serializer_class(data=request.data)
+        return self.post(self, request, *args, **kwargs)
+
+    @swagger_auto_schema(request_body=MessageSerializer, responses={200: MessageSerializer})
```

Comment:
For me, it only works if add the `swagger_auto_schema` line to the `create` method.